### PR TITLE
Feature/#345 more information in access request email

### LINKED ISSUE #345: Include more information in access request mail
Include
* User's social identities (e.g. OpenID IDs)
* User's full name
* Deep link to user in admin interface

### PR DESCRIPTION
**Pull request by das-g**
closes #345 

reviewed by:
- [x] @hixi

### CODE REVIEW

File: web_frontend/osmaxx/excerptexport/views.py [hunk 1]
```diff
@@ -11,6 +11,8 @@
 from django.utils.translation import ugettext_lazy as _
 from django.views.generic import FormView
 from django.conf import settings
+
+from config.settings.common import OSMAXX_FRONTEND_USER_GROUP
```
Comment (hixi) [inline web_frontend/osmaxx/excerptexport/views.py]: don't import settings like this. Please use the overridable variant as stated in the [django documentation](https://docs.djangoproject.com/en/1.9/topics/settings/#using-settings-in-python-code).

**Comment by hixi** [comment]
besides the one comment this looks good to me.

File: web_frontend/osmaxx/excerptexport/views.py (hunk 1 above)
Comment (das-g) [inline web_frontend/osmaxx/excerptexport/views.py]: like so?  2d8410d

Comment (hixi) [inline web_frontend/osmaxx/excerptexport/views.py]: Yes

**Pull request merged**